lib/Ska/Web.pm

```diff
@@ -121,10 +121,11 @@ sub get_html_content {
 	my $uri = URI->new($src);
 	my $img_uri = $uri->abs($base_uri);
 	next if $img_uri =~ /$dummy_url/; # No base URL supplied and src is relative
-	my ($image_data, $error) = get_url($img_uri);
+	my ($image_data, $error, $header) = get_url($img_uri);
 	push @images, {data => $image_data,
 		       name => $img_uri->rel($img_uri)->as_string,
 		       url  => $img_uri->as_string,
+                       header => $header,
 		      } unless defined $error;
```

Comment:
Unrelated, but having been away from Perl for a while it seems like `unless` is a really bad language construct, esp. in the context of a long-ish code statement like this.  I was looking at this for a bit wondering why `$error` was being ignored.